### DISCUSSION TITLE
Resume_collection() warning and failure

### DISCUSSION THREAD

**Discussion by shmuhammadd** [cjbarrie/academictwitteR#217] [General]
Hi, I collected tweets as shown below :

```
get_all_tweets(
  query = happy,
  start_tweets = "2016-01-01T00:00:00Z",
  end_tweets = "2021-08-10T00:00:00Z",
  data_path = "happy",
  bind_tweets = F,
  n = Inf,
  bearer_token = bearer_token,
  country = "ng"
)
```

The collection fails along the way with the error below:

```
Error in make_query(url = endpoint_url, params = params, bearer_token = bearer_token,  : 
  Too many errors.
```

I  tried to update the collection with: 
 
 `resume_collection(data_path = "happy")`

But, I receive warning only:

```
Warning messages:
1: Tweets will be bound in local memory as well as stored as JSONs. 
2: Directory already exists. Existing JSON files may be parsed and returned, choose a new path if this is not intended. 
```

Please, advise.

Thank you.

**Comment by chainsawriot** [comment]
@shmuhammad2004 It's possible that all tweets that could be collected have already been collected. Please try to `bind_tweets` your "happy" directory and see the oldest `created_at`.

**Reply by NicuSV** [reply]
@chainsawriot This is unlikely the cause; after I run `resume_collection(data_path = path)`, the tweets update, but I have to run it manually every time (I can see this also with `bind_tweets`). However, the tweets are only for two days (the last two days) -- this is something that I also do not understand: they are not collected chronologically? Suppose that there are in total 10M tweets from t(0) to t(n) and I want to collect 1M from t(0) to t(n-k) -- I will get 1M tweets proportionally to my time request, or only from t(0) to t(n-k-r)? Thanks.

**Reply by chainsawriot** [reply]
@NicuSV If you go deeper into the documentation of `resume_recollection`, you'll notice the argument `...`. From there, you will know that the `...` denotes additional arguments to `get_all_tweets`. One thing that you can try, is to override the default `n` parameter (which is 100). i.e.

```R
resume_collection(data_path = path, n = Inf)
```

As a general advice, try to break down your long query into small ones. We can't test long queries.
